docs/release.md

```diff
@@ -0,0 +1,55 @@
+# Status AddOn Relesea process 
+
+The Status AddOn release process is based on [GoReleaser](https://goreleaser.com).
+GoReleaser is configured to automatically create a new release by pushing new tags matching the
```

Comment:
The wording here is a bit off. "GoReleaser is configured to automatically create a new release by pushing new tags..." says that GoReleaser will accomplish its release-creating job by pushing new tags. I think that is not what you are trying to say. I think you are trying to say that GoReleaser is configured to create a new release _in response to_ somebody (anybody) pushing new tags.

Reply:
That makes sense, but can anybody push tags to our repo ? I replaced somebody/anybody with maintainers

Reply:
This is now moot as I am ok with your proposed rewrite of the paragraph below.